Source/Core/VideoBackends/OGL/FramebufferManager.cpp

```diff
@@ -99,8 +99,8 @@ FramebufferManager::FramebufferManager(int targetWidth, int targetHeight, int ms
 
     glBindTexture(m_textureType, m_efbDepth);
     glTexParameteri(m_textureType, GL_TEXTURE_MAX_LEVEL, 0);
-    glTexImage3D(m_textureType, 0, GL_DEPTH_COMPONENT32F, m_targetWidth, m_targetHeight,
```

Comment:
My suggestion is to use two textures, one for the depth, one for the stencil. So they aren't interleaved and it should be close to zero overhead to only write the depth. It would also simplify the readback logic with glGetTexImage as there is a single texture just for stencil.